lib/events.ts

```diff
@@ -184,7 +188,6 @@ export interface WaitOptions {
 	id: string;
 	actor: string;
 	action?: string;
-	card?: string;
```

Comment:
As far as I searched in our repositories, I found no use of this property in other packages.